games/a/pommerman/envs/v0.py

```diff
@@ -433,11 +433,9 @@ def render(self, mode='human', close=False, record_pngs_dir=None, record_json_di
                 self._viewer = None
             return
 
-        if record_pngs_dir:
-            assert not os.path.isdir(record_pngs_dir)
+        if record_pngs_dir and not os.path.isdir(record_pngs_dir):
             os.makedirs(record_pngs_dir)
-        if record_json_dir:
-            assert not os.path.isdir(record_json_dir)
+        if record_json_dir and not os.path.isdir(record_json_dir):
```

Comment:
if the directories already exist, then this is going to silently keep writing to them[, and possibly overriding]. that seems bad.